tests/pygame_gui/test_components.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+from unittest.mock import MagicMock
+import pygame
+import sys
+import os
+
+print(os.path.dirname(__file__))
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '../..')))
+
+from src.pygame_gui.components.root import *
```

Comment:
Import concrete class. For examle
`from src.pygame_gui.components.root import Root`
You should never use `import *`